distributionviewer/core/static/css/secondary-menu.styl

```diff
@@ -7,36 +7,43 @@ $my-height = 35px;
   border-top: 1px solid $dark-blue;
   display: flex;
   height: $my-height;
-  justify-content: flex-end;
+  justify-content: flex-end; /* So that the gear appears to the very right */
   left: 0;
   padding: 0 30px;
   position: fixed;
   bottom: 0;
   width: 100%;
   z-index: 10;
-
-  > * {
-    align-self: center;
-  }
 }
 
 .configure-link, .configure-link img {
   height: 25px;
   width: 25px;
+  align-self: center;
 }
 
 .secondary-menu-content {
   display: none;
   height: $my-height;
-  justify-content: center;
   margin: 0 auto;
-  width: 600px;
+  font-size: 12px;
+  font-weight: 100;
+  line-height: $my-height;
```

Comment:
`$my-height` is `$my-fault` but months later I forgot what it represents. You don't have to fix it but I should know better by now.